refarch-integrations/refarch-email-integration/refarch-email-integration-rest-example/src/main/resources/application-local.yml

```diff
@@ -6,6 +6,25 @@ spring:
     port: 1025
     username: test@muenchen.de
     password: secret
+  security:
+    oauth2:
+      client:
+        provider:
+          keycloak:
```

Comment:
the provider name keycloak should not be product specific.

An alternative would be sso or an other appropriate name.

Reply:
Done